packages/mysql/package.json

```diff
@@ -36,9 +36,9 @@
     "@sequelize/core": "workspace:*",
     "@sequelize/utils": "workspace:*",
     "dayjs": "^1.11.10",
-    "lodash": "4.17.21",
-    "mysql2": "3.9.2",
-    "wkx": "0.5.0"
+    "lodash": "^4.17.21",
+    "mysql2": "^3.9.2",
+    "wkx": "^0.5.0"
   },
   "devDependencies": {
```

Comment:
These devDependencies were added in error and can be removed.

Reply:
Indeed, because they do not run their own unit tests. Removed from the applicable packages